ara/static/css/ara.css

```diff
@@ -80,4 +80,12 @@ td a {
 
 .list table, .list th, .list td {
     text-align: center;
-}
\ No newline at end of file
+}
+
+.incomplete {
```

Comment:
Oh, yuck, that's an icon font.  That's a bit of a hack (problems with font downloading result in goofy display problems).